main.cpp

```diff
@@ -131,6 +256,26 @@ struct RockClimber
     bool completeCheck ( int difficulty, double experience, double strength );
 };
 
+void RockClimber::climb( BoulderProblem blue, TopRopeRoute red, CrackClimb green )
+{
+    bool x = false, y = false, z = false;
+
+    if( experience * blue.problemGrade >= strength )
+    {
+        x = true;
+    }
+
+    if( experience / red.wallAngle > 5 )
+    {
+        y = true;
+    }
+
+    if( strength / green.crackWidth >= experience )
+    {
+        z = true;
+    }
```

Comment:
You can initialize your boolean variables with much more simplicity: 
```
bool x = experience * blue.problemGrade >= strength;
```
- [ ] simplify variable initialization.